research/H1/Sublabel_DF.md

```diff
@@ -0,0 +1,179 @@
+# Label: H1, Sublabel: DF
```

Comment:
Where are you getting the sub labels from? Which client and can you post a full log of that?

Reply:
I am using dumpvdl2.    I log the messages as JSON locally.   Often I don't log locally and only send them to Airframes.io, due to current equipment limitations.

Here is a fragment of an H1, sub DF message I collected earlier today.

"acars": {
      "err": false,
      "crc_ok": true,
      "more": false,
      "reg": ".N27253",
      "mode": "2",
      "label": "H1",
      "blk_id": "3",
      "ack": "!",
      "flight": "UA0614",
      "msg_num": "D41",
      "msg_num_seq": "B",
      "sublabel": "DF",
      "msg_text": "ABS011DA_N27253  KIAHKSMF4\r\n 30018 -953982310000761+308125016XH08GX1151420000--\r\n 29930 -953620155001989+24-143014X1P/9W0661150000--\r\n 29907 -954310156004131+21-111011XZ:700++++++0000--\r\n 29918 -955200157006476+165 98011XZ:700++++++0000--\r\n 29942 -956100158008966+14-112010XZ:700++++++0000--\r\n 29967 -957060159010665+12- 98007XZ:700++++++0000--\r\n"
    }


I will go back and find the original message(s).